app/views/teachers/_teacher_info.html.erb

```diff
@@ -1,68 +1,106 @@
 <div class="card">
-  <div class="card-header">
-    <div class="row d-flex">
-      <div class="col-sm-8 d-flex">
-        <h3>
-          <%= teacher.full_name %>
-        </h3>
-      </div>
-      <div class="col-sm-4 text-right">
-        <%= link_to "Edit Information", edit_teacher_path(teacher), class: "btn btn-primary" %>
-        <%= link_to("Delete", teacher_path(teacher), method: "delete", class: "btn btn-danger", data: {confirm: "Are you sure?"}) %>
-      </div>
+  <div class="card-header d-flex justify-content-between align-items-center">
+    <h3><%= teacher.full_name %></h3>
+    <div>
+      <%= link_to "Edit Information", edit_teacher_path(teacher), class: "btn btn-primary mr-2" %>
+      <%= link_to("Delete", teacher_path(teacher), method: "delete", class: "btn btn-danger", data: {confirm: "Are you sure?"}) %>
     </div>
   </div>
-  <div class="card-body ml-5">
-    <div class="row mt-3 pb-3">
-      <div class="col-sm-4">
-        <h5>Email</h5>
+  <div class="card-body">
+    <div class="row mb-3">
+      <div class="col-sm-4 font-weight-bold">Email:</div>
+      <div class="col-sm-6 text-center">
         <a href=<%= "mailto:#{teacher.email}" %>> <%= teacher.email %> </a>
       </div>
-
-      <div class="col-sm-4">
-        <h5>Snap<i>!</i> Username</h5>
+      <div class="col-sm-2">
+        <button class="btn btn-primary" onclick="copyToClipboard('<%= teacher.email %>')">Copy</button>
```

Comment:
Can we use a `btn-sm` or `btn-xs` class? And instead of copy, I think We should have font awesome, and I think we should be able to use `fa-copy`